### DISCUSSION TITLE
Hosts that need to be whitelisted

### DISCUSSION THREAD

**Discussion by T145** [T145/black-mirror#57] [Ideas]
Recommend hosts to be whitelisted, such as false positives.

**Comment by iam-py-test** [comment]
@T145 
`whatsapp.net`: https://github.com/TheAntiSocialEngineer/AntiSocial-BlockList-UK-Community/issues/11
`live.net`: https://github.com/TheAntiSocialEngineer/AntiSocial-BlockList-UK-Community/issues/12

**Reply by T145** [reply]
`live.net` is dead, and `whatsapp.net` is intended to be blocked. If you want secure mobile conversations, services like Signal are far better and do not have a track record of having numerous vulnerabilities and breaching privacy rights.

**Reply by iam-py-test** [reply]
> `live.net` is dead

That is weird, but if it is, then there is really no reason to allowlist it

`whatsapp.net` is intended to be blocked. If you want secure mobile conversations, services like Signal are far better and do not have a track record of having numerous vulnerabilities and breaching privacy rights.

I’m not sure I understand
This block seems to be breaking a legitimate service
I understand that it is not secure or private, but I would think that is the user’s choice.

**Reply by T145** [reply]
Black Mirror's purpose is to aggressively promote its [goals](https://github.com/T145/black-mirror#-goals). Hence why it blocks more content than any other popular blacklist. Whatsapp falls into those parameters, while something like Signal does not.

I may need to take some time and rewrite the guidelines to communicate the ideals thoroughly.

**Comment by T145** [comment]
May be some hosts from https://github.com/MISP/misp-warninglists
